aws-java-sdk-dynamodb/pom.xml

```
@@ -4,7 +4,7 @@
   <parent>
     <groupId>com.amazonaws</groupId>
     <artifactId>aws-java-sdk-pom</artifactId>
-    <version>1.11.762-dw4-SNAPSHOT</version>
+    <version>1.12.138-dw-SNAPSHOT</version>
```

```suggestion
    <version>1.12.138-dw1-SNAPSHOT</version>
```
This is so Maven release can manage the minor version automatically and bump it to `1.12.138-dw2-SNAPSHOT`